app/src/main/java/com/babylon/wallet/android/presentation/dapp/authorized/account/AccountSelectionCard.kt

```diff
@@ -68,19 +65,22 @@ fun AccountSelectionCard(
                 selected = checked,
                 colors = RadioButtonDefaults.colors(
                     selectedColor = RadixTheme.colors.gray1,
-                    unselectedColor = RadixTheme.colors.gray3,
-                    disabledSelectedColor = Color.White
+                    unselectedColor = RadixTheme.colors.white,
+                    disabledSelectedColor = RadixTheme.colors.white
                 ),
                 onClick = radioButtonClicked,
                 enabled = isEnabledForSelection
             )
         } else {
             Checkbox(
                 checked = checked,
-                colors = CheckboxDefaults.colors(
-                    checkedColor = RadixTheme.colors.gray1,
-                    uncheckedColor = RadixTheme.colors.gray3,
-                    checkmarkColor = Color.White
+                colors = CheckboxDefaults.colors().copy(
+                    checkedCheckmarkColor = RadixTheme.colors.gray1,
+                    checkedBorderColor = RadixTheme.colors.white,
+                    checkedBoxColor = RadixTheme.colors.white,
+                    uncheckedCheckmarkColor = Color.Transparent,
+                    uncheckedBorderColor = RadixTheme.colors.white,
+                    uncheckedBoxColor = RadixTheme.colors.white.copy(alpha = 0.5f)
```

Comment:
Hey, awesome that you did that change!

Reply:
We also need to check on radio buttons' theming. You can test a radio button by using sandbox and require exactly 1 account.

Reply:
I wanted to do the same for Radio Buttons but unfortunately the colors we want to change are not exposed and the only way to make them look like in our design is to define our own completely custom radio buttons. Is it worth it, what do you think?

Reply:
Well it is worth creating a ticket for it. I remember trying to do the same in the past.

Reply:
I'll take care of it then.